src/renderTask.js

```diff
@@ -0,0 +1,19 @@
+import Task from './taskclass.js';
+
+const showPage = (taskList) => {
+  const renderPage = document.querySelector('.renderPage');
+
+  renderPage.innerHTML = '';
+  taskList.map((task, index) => {
+    // taskList.sort((a, b) => a - b);
```

Comment:
- [ ] In order to make your codebase cleaner I suggest you delete commented code on line 8.